composer.json

```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^7.1.8",
+        "ext-json": "*",
```

Comment:
learned just recently that it's not needed for php>5.2 (unless you compiled php specifically without json)

Reply:
> learned just recently that it's not needed for php>5.2 (unless you compiled php specifically without json)

This is not strictkly requirement, although some linting or deployment tools requires this to build the php image accordingly.

Although I do agree with you.